File: 240918/개수 세기/count-numbers.c
#include <stdio.h>

int main() {
    int n,m,i,k,cnt=0;

    scanf("%d %d", &n,&m);
    int arr[n];
    for(i=0;i<n;i++){
        scanf("%d ", &arr[i]);
        if(arr[i]==m){
            cnt+=1;
        }
    }
    printf("%d", cnt);
    return 0;
}

File: 250306/개수 세기/count-numbers.py
# 정수 n과 m을 입력받습니다.
n, m = tuple(map(int, input().split()))

# arr을 입력받습니다.
arr = list(map(int, input().split()))

# m이 나온 횟수를 출력합니다.
cnt = arr.count(m)
print(cnt)